src/Typedefs.idr

```diff
@@ -156,3 +156,20 @@ mutual
 
 Show (TDef n) where
   show = showTDef
+
+-- Equality -----
```

Comment:
In keeping with the rest of this file, which is pretty much the main `Typedefs` thing, I propose to align on `==`. This file is pretty central to Typedefs so it seems like a good place for a bit of polish.